### ISSUE TITLE
Add word count for *all* languages, not just base/source language

### ISSUE THREAD

**Issue by jdimeo** [tolgee/tolgee-platform#1624]
When billing translators or measuring progress, it's very helpful to see the current word count for ALL languages in a project not just the base/source language. Some of our projects don't even have a source language (see #1623 ). If we're calculating word stats in the background, can we do that for all languages and display that say on the project's language page? (https://app.tolgee.io/projects/###/languages)

**Comment by JanCizmar** [comment]
Thinking about this might be solved with the ability to select source language for each language instead of having single base. https://github.com/tolgee/tolgee-platform/issues/1391

**Comment by jdimeo** [comment]
Yes, that would help a lot! That's even better than not supporting a base language, but rather having many base languages (`en-from-es-ES` would have base language `es-ES`)

**Comment by jdimeo** [comment]
Please keep this open!
